ios/test_app.rb

```diff
@@ -91,8 +91,8 @@ def resources_pod(project_root, target_platform)
   app_manifest = find_file('app.json', project_root)
   return if app_manifest.nil?
 
-  resources = resolve_resources(JSON.parse(File.read(app_manifest)), target_platform)
-  return if !resources.instance_of?(Array) || resources.empty?
+  manifest_resources = resolve_resources(JSON.parse(File.read(app_manifest)), target_platform)
+  resources = ['app.json'] + (manifest_resources.instance_of?(Array) ? manifest_resources : [])
```

Comment:
What else could `manifest_resources` be and should it warrant throwing an error?

Reply:
It could be `nil`, e.g. because it is missing in the manifest, and that is fine.

Reply:
Gotcha. Just FYI, a more idiomatic way could be to use the ‘splat’ operator:

```
irb(main):001:0> ["app.json", *nil]
=> ["app.json"]
irb(main):002:0> ["app.json", *["foo"]]
=> ["app.json", "foo"]
```

Reply:
Ooh, nice. I'll try it out next time I submit a PR 😄